src/install/postgresql/whatsNext.mdx

```diff
@@ -0,0 +1,1182 @@
+---
+componentType: default
+headingText: What's next?
+---
+
+## PostgreSQL users and permissions [#create-user]
+
+Create a user with `SELECT` permissions on:
```

Comment:
I'm a bit confused about this section. It seems to be a mix of procedural and reference material. Also, some of the steps above tell people to create a user and link to this section.

I think this specific section needs a bit of a rewrite to pull out the procedural material from the reference material. The reference material should stay here, and the procedural material should be included as a separate step above.